docs/html/reference/pip_install.rst

```diff
@@ -858,6 +865,13 @@ Examples
       $ pip install http://my.package.repo/SomePackage-1.0.4.zip
 
 
+#. Install a particular source archive file following :pep:`440` direct references.
+
+    ::
+
+      $ pip install SomeProject==1.0.4@http://my.package.repo//SomeProject-1.2.3-py33-none-any.whl
```

Comment:
Do you need spaces around the `@` symbol here? (And also command-line quoting for the argument.)

Reply:
You don't need spaces, but if you use it you need to quote it. I'll add it to the example.